request-management-api/request_api/models/FOIRawRequests.py

```diff
@@ -47,13 +65,21 @@ def updateworkflowinstance(cls,wfinstanceid,requestid)->DefaultMethodResult:
     @classmethod
     def getrequests(cls):
         request_schema = FOIRawRequestSchema(many=True)
-        query = db.session.query(FOIRawRequest).all()
-        return request_schema.dump(query)
+        _session = db.session
+        _requestids = _session.query(distinct(FOIRawRequest.requestid)).all()
+        requests = []
+        for _requestid in _requestids:
+           request = _session.query(FOIRawRequest).filter(FOIRawRequest.requestid == _requestid).order_by(FOIRawRequest.version.desc()).first()           
+           requests.append(request)
+        #query = db.session.query(FOIRawRequest).all()
+        #query = db.session.query(FOIRawRequest).distinct(FOIRawRequest.requestid).all()
+        #return request_schema.dump(query)
```

Comment:
Random thought: Could put things like dumping the request query behind logging environment variables, or checking dev environment or something.  Only if you'd find that useful.